tests/test_scrape.py

```diff
@@ -1,32 +1,93 @@
+import collections
+
 import pytest
 
-from wikipron.scrape import scrape
+from wikipron.scrape import scrape, _skip_word, _skip_date
+from wikipron.extract import EXTRACTION_FUNCTIONS
 
 from . import can_connect_to_wiktionary, config_factory
 
 
-@pytest.mark.skipif(not can_connect_to_wiktionary(), reason="need Internet")
-@pytest.mark.parametrize(
-    "key, language, other_params",
-    [
-        ("eng", "English", {}),
-        # Test that 'sup[a[@title = "wikipedia:Slovak phonology"]]' works.
-        ("slk", "Slovak", {}),
-        # Test that the extra "span" layer for Korean is handled.
-        # Korean data is mostly phonetic transcription only.
-        ("kor", "Korean", {"phonetic": True}),
-    ],
+SmokeTestLanguage = collections.namedtuple(
+    "SmokeTestLanguage", ("key", "wik_name", "config_params")
 )
+SmokeTestLanguage.__doc__ = """
+Represents a language to run a smoke test on.
+
+Parameters
+----------
+key : str
+    An ISO 639 code or language name.
+wik_name : str
+    The language name used by Wiktionary.
+config_params : dict
+    Parameters for the Config class.
+"""
```

Comment:
I'm used to the [numpydoc format](https://numpydoc.readthedocs.io/en/latest/format.html) for docstrings simply because I'm in the data science world now with extensive use of the pydata stack (sklearn, etc.) all documented in this style. If you guys feel strongly about using another docstring protocol, I'd be happy to comply. (IMO, numpydoc is clearer than the (default?) reStructuredText format.)

Reply:
I am fine with this entirely, so long as we standardize. (And I think we should add more typing too.)